.github/workflows/docker-publish.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     continue-on-error: false
     env:
       TAG: github-1
+    environment:
```

Comment:
IIUC the bouncer bot will gate this if my PR changes the trigger to pull_request in line 4 above?

Reply:
this job only runs on pushs, but sorta.  the bouncer bot will only allow builds if they are from a push to protected branch by bors-libra.   a push by you to one of the branches above wouldn't get secrets.